src/blueapi/rest/handler.py

```diff
@@ -0,0 +1,77 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+from blueapi.config import ApplicationConfig, AppSettings
+from blueapi.core import BlueskyContext
+from blueapi.messaging import StompMessagingTemplate
+from blueapi.messaging.base import MessagingTemplate
+from blueapi.utils import ConfigLoader
+from blueapi.worker.reworker import RunEngineWorker
+from blueapi.worker.worker import Worker
+
+settings = AppSettings()
+
+
+class Handler:
+    context: BlueskyContext
+    worker: Worker
+    config: ApplicationConfig
+    message_bus: MessagingTemplate
```

Comment:
Should we allow message_bus to be null?

Reply:
This isn't a dataclass, so my understand is it's never really null - when it's initialised it's set to something. It needs to be set in __init__ because it needs configuration...

Thinking about it, now that I've made the config file path an environment variable that is ingested by `pydantic.BaseSettings`, it might be a good idea to simplify the `__init__` function here and remove the config_path entirely, instead taking it from the settings. But that's if you like what I've done with the settings - we could, potentially, revert back to config being passed via the CLI. I'd probably need to do some research into how to pass args to fastAPI but I think it should be possible.

Reply:
I think Joseph means should we permit the existence of a handler with no message bus? You currently force it to exist. I think the answer is "yes, but possibly out of scope for this change".

Reply:
Hmm, okay. Perhaps something to do for a future PR?

Reply:
Yeah exactly, maybe make a new issue about it